cfpb/sbl-frontend
# Design Review - style(App): Fixes having the unnecessary whitespace below the FooterCfGov

closes https://github.com/cfpb/sbl-frontend/issues/365

## Changes
- style(App): Fixes having the unnecessary whitespace below the FooterCfGov

## How to test this PR

1. `yarn install` to get the latest DSR update
2. Go to any page where FooterCfGov is used (basically all)
3. Use mobile view and swipe up at the scroll end OR zoom out (CMD + - multiple times)
4. See how the footer's grey part still extends to the bottom

## Screenshots BEFORE the fix
![Screenshot 2024-04-02 at 12 29 43 PM](https://github.com/cfpb/sbl-frontend/assets/13324863/72228a6a-508d-428c-a1ff-a54f1ef81aeb)
![Screenshot 2024-04-02 at 12 31 41 PM](https://github.com/cfpb/sbl-frontend/assets/13324863/56bc509b-b0d4-4fce-a074-c83f6c810122)

## Screenshot - Fix B (View zoomed out) - Maximizing the footer 
<img width="1552" alt="Screenshot 2024-04-02 at 10 15 16 PM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/c686516a-978d-451f-b6d3-26930ca808ce">
